proto/protovalidate/buf/validate/validate.proto

```diff
@@ -3325,8 +3341,15 @@ message StringRules {
       }
     ];
 
-    // `ip` specifies that the field value must be a valid IP
-    // (v4 or v6) address, without surrounding square brackets for IPv6 addresses.
+    // `ip` specifies that the field value must be a valid IP (v4 or v6) address.
+    //
+    // IPv4 addresses are expected in the dotted decimal format, for example "192.168.5.21".
+    // IPv6 addresses are expected in their text representation, for example "::1",
+    // or "2001:0DB8:ABCD:0012::0".
+    //
+    // Both formats are well-defined in the internet standard RFC 3986. Zone
```

Comment:
my only nit with these changes is that we're now inconsistent about linking out to these documents (most of the RFCs). I think we ought to decide on a consistent approach w.r.t. linking out. I'm assuming not linking was deliberate?

Reply:
Not completely sure that they are helpful, but I didn't intend to break the convention. Added links in https://github.com/bufbuild/protovalidate/pull/320/commits/4173eb86655d510f18a5b6fbb85781dbf879a326.